examples/real_function_space.py

```diff
@@ -168,7 +216,7 @@ def u_exact(x):
 
 import pyvista
 
-pyvista.start_xvfb()
+pyvista.start_xvfb(1.0)
```

Comment:
What does `1.0` do here?

Reply:
Waiting time for xvfb to start, default is 5 seconds.